app/src/org/commcare/activities/CommCareWiFiDirectActivity.java

```diff
@@ -822,6 +894,10 @@ public CustomProgressDialog generateProgressDialog(int taskId) {
                 title = localize("wifi.direct.wipe.task.title").toString();
                 message = localize("wifi.direct.wipe.task.message").toString();
                 break;
+            case FormRecordToFileTask.PULL_TASK_ID:
+                title = localize("wifi.direct.pull.task.title").toString();
+                message = localize("wifi.direct.pull.task.message").toString();

```

Comment:
What is the difference between doing a `localize().toString()` and a `Localization.get()`? Does the first format markup and then discard it in the `toString`? If so we should consider encapsulating that logic in a documented method.